sdk/formrecognizer/Azure.AI.FormRecognizer/src/FormRecognizerClient.cs

```diff
@@ -60,10 +59,9 @@ public FormRecognizerClient(Uri endpoint, AzureKeyCredential credential, FormRec
 
             _endpoint = endpoint;
             _credential = credential;
-            _options = options.Clone();
```

Comment:
Clone removed for two reasons:
- Recordings wouldn't work otherwise, since options must be instrumented.
- We were not cloning the whole object anyway (we cloned the `Version` only, but missed other members inherited from `ClientOptions` that couldn't be cloned).

Since the only purpose of the options is creating the `ServiceClient`, I created an `internal` constructor in `FormTrainingClient` that takes a `ServiceClient` instance. This way, this `FormRecognizerClient` and a training client created from it will share the same configuration.